extensions/blocks/eventbrite/edit.js

```diff
@@ -0,0 +1,397 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { __, _x } from '@wordpress/i18n';
+import { Component } from '@wordpress/element';
+import {
+	Placeholder,
+	SandBox,
+	Button,
+	IconButton,
+	Toolbar,
+	PanelBody,
+	Spinner,
+	ExternalLink,
+} from '@wordpress/components';
+import { BlockControls, BlockIcon, BlockPreview } from '@wordpress/block-editor';
+import { withDispatch } from '@wordpress/data';
+import { InspectorControls } from '@wordpress/editor';
+import apiFetch from '@wordpress/api-fetch';
+import { ENTER, SPACE } from '@wordpress/keycodes';
+import { getBlockFromExample } from '@wordpress/blocks';
+/**
+ * Internal dependencies
+ */
+import { createWidgetId, convertToLink, eventIdFromUrl } from './utils';
+import { CUSTOM_URL_REGEX, icon, URL_REGEX } from '.';
+import ModalButtonPreview from './modal-button-preview';
+import EventbriteInPageExample from './eventbrite-in-page-example.png';
+import './editor.scss';
+
+const MODAL_BUTTON_STYLES = [
+	{ name: 'fill', label: __( 'Fill' ), isDefault: true },
```

Comment:
You'll probably want to make those strings translatable. That's not a blocker though, can be done later.

Reply:
Tracked in https://github.com/Automattic/jetpack/issues/14458